src/demo-app/select/select-demo.html

```diff
@@ -3,7 +3,7 @@
 <div class="demo-select">
   <div *ngIf="showSelect">
     <md-card>
-      <md-select placeholder="Food i would like to eat" [formControl]="foodControl">
+      <md-select placeholder="Food i would like to eat" [formControl]="foodControl" (change)="changeListener($event)">
```

Comment:
Can we make a separate demo for the change event that does not use a form directive? The change event is only intended for people not using Angular forms modules, since these already have their own built-in change events. Just don't want to confuse the intended usage with this example.